project/src/data/new-evaluations.js

```diff
@@ -0,0 +1,297 @@
+            },
+            {
+                label: "Exception Handling",
+                options: optionsItems
+            },
+            {
+                label: "Version Control",
+                options: optionsItems
+            }
+        ],
+        [
+            {
+                label: "Access Modifiers",
+                options: optionsItems
+            },
+            {
+                label: "Design Patterns",
+                options: optionsItems
+            },
+            {
+                label: "Issue Tracking",
+                options: optionsItems
+            }
+        ],
+        [
+            {
+                label: "Polymorphism",
+                options: optionsItems
+            },
+            {
+                label: "RegEx",
+                options: optionsItems
+            }
+    ]
+    ];
+    const dropdownOptions2 = [
+        [
+            {
+                label: "Protocol",
+                options: optionsItems
+            },
+            {
+                label: "Response Codes",
+                options: optionsItems
+            },
+            {
+                label: "Rest",
+                options: optionsItems
+            }
+        ],
+        [
+            {
+                label: "Headers",
+                options: optionsItems
+            },
+            {
+                label: "Request Methods",
+                options: optionsItems
+            },
+            {
+                label: "Sessions & Cookies",
+                options: optionsItems
+            }
+        ]
+    ];
+    const dropdownOptions3 = [
+        [
+            {
+                label: "Doctype",
+                options: optionsItems
+            },
+            {
+                label: "Tags",
+                options: optionsItems
+            },
+            {
+                label: "Basic SEO",
+                options: optionsItems
+            }
+        ],
+        [
+            {
+                label: "Syntax Rules",
+                options: optionsItems
+            },
+            {
+                label: "Attributes",
+                options: optionsItems
+            }
+        ]
+    ];
+    const dropdownOptions4 = [
+        [
+            {
+                label: "Usage",
+                options: optionsItems
+            },
+            {
+                label: "Box modeling",
+                options: optionsItems
+            },
+            {
+                label: "CSS 3.0",
+                options: optionsItems
+            }
+        ],
+        [
+            {
+                label: "Selectors",
+                options: optionsItems
+            },
+            {
+                label: "Styling",
+                options: optionsItems
+            },
+            {
+                label: "Dynamic Stylesheets",
+                options: optionsItems
+            }
+        ]
+    ];
+    const dropdownOptions5 = [
```

Comment:
Can you explain why you need the nested arrays throughout these `dropdownOptions*` variables?

Reply:
It's complicated.... When I've built the HTML I've considered putting each row of each fieldset in a li tag and it was pretty hard to find a proper data structure for sending it as parameter. I discussed with Gabi yesterday about it and he gave me the idea of that nested array. I know it's a more complicated than sending a simple array but at least I've learned how to approach such a situation and to think twice the next time I write HTML.